apps/docs/pages/guides/platform/network-restrictions.mdx

```diff
@@ -14,6 +14,10 @@ Network Restrictions is currently in beta and is slowly being made available to
 
 Each Supabase project comes with configurable restrictions on the IP ranges that are allowed to connect to Postgres and its pooler ("your database"). These restrictions are enforced before traffic reaches your database. If a connection is not restricted by IP, it still needs to authenticate successfully with valid database credentials.
 
+If direct connections to your database [resolve to a IPv6 address](https://supabase.com/dashboard/project/_/settings/database), you need to add both IPv4 and IPv6 CIDRs to the list of allowed CIDRs. IPv4 CIDRs will be used to restrict connections to the database pooler, while IPv6 CIDRs will be used to restrict connections to Postgres. [Supavisor](https://github.com/supabase/supavisor/) does not currently support Network Restrictions and will start enforcing restrictions from 19th January 2024.
```

Comment:
makes it sound like ipv6 restrictions wont apply for supavisor and similarly ipv4 restrictions wont apply to db box

Reply:
both restrictions are applied to both products